subprojects/shipkit/src/test/groovy/org/shipkit/internal/gradle/init/InitPluginIntegTest.groovy

```diff
@@ -0,0 +1,30 @@
+package org.shipkit.internal.gradle.init
+
+import org.gradle.testkit.runner.BuildResult
+import testutil.GradleSpecification
+
+class InitPluginIntegTest extends GradleSpecification {
+
+        def "runs initShipkit task in a project without any Shipkit configuration"() {
+            given:
+            projectDir.newFolder("gradle")
```

Comment:
I don't think this is needed. If "gradle" dir does not exist, initialization should work.